core/src/main/java/org/infinispan/remoting/inboundhandler/AbstractDelegatingHandler.java

```diff
@@ -0,0 +1,49 @@
+package org.infinispan.remoting.inboundhandler;
+
+import org.infinispan.commands.remote.CacheRpcCommand;
+
+/**
+ * Designed to be overwrite.
```

Comment:
I assume you mean overridden, but this is an abstract class so it doesn't say anything new...